routes/api/v1/foods.js

```diff
@@ -28,4 +28,18 @@ router.get('/:id', (req, res) => {
   })
 });
 
+router.post("/", function(req, res, next) {
```

Comment:
```suggestion
router.post("/", function(req, res) {
```
Generally, if we aren't using it, don't include it.